### PR TITLE
remove openshift-must-gather inspect command

### PR DESCRIPTION
**Pull request by sanchezl**
Replaced in https://github.com/openshift/oc/pull/107 by `oc adm inspect`.
Replace `openshift-must-gather` binary with script that forwards calls to `oc adm inspect`.

### CODE REVIEW

**Comment by sanchezl** [comment]
/hold 

until https://github.com/openshift/oc/pull/42 merges

File: collection-scripts/gather [hunk 1]
```diff
@@ -28,8 +28,8 @@ resources+=(clusternetworks hostsubnets)
 resources+=(clusterautoscaler machineautoscaler)
 
 # Run the Collection of Resources using must-gather
```
Comment (sallyom) [inline collection-scripts/gather]: nit: fix the comment

**Pull request merged**